src/tetris/biasEngine/biasEngine.ts

```diff
@@ -48,21 +49,20 @@ export default class BiasEngine {
 		biasProfileWeights.set(BiasEngine._calculateOperatingSystemBias, 0.15);
 		return biasProfileWeights;
 	}
+
+    // @ts-ignore
```

Comment:
This is, because the `getter` and the `setter` for `currentDetectionLevel` dont have the same visibility (and this is intended, since only the `BiasEventGenerator` should be able to change that value, while everyone can see it)
Sadly, this kind of scenario seems to be considered "bad code" (such producing a warning) in typescript.

Reply:
😆